src/config/route-middlewares.js

```diff
@@ -17,9 +17,15 @@ module.exports = {
         req.simulatorSocket = req.socketConnections[`${publicKey}-${device}`];
 
         if (!req.simulatorSocket) {
-            return utils.errorResponse(res, [{ msg: constatnts.errorMessages.deviceNotConnetcted }]);
+            return utils.errorResponse(res, [{ msg: constants.errorMessages.deviceNotConnetcted }]);
         }
 
         next();
+    },
+
+    healthcheckMiddleware: (req, res, next) => {
+        res.status(constants.responseCode.ok)
+            .json({ status: constants.statusMassages.OK })
+            .end();
```

Comment:
I think you needn't call `end` here because `json` calls `send` for you and `send` calls `end` on the `response`